qutip/core/gates.py

```diff
@@ -52,13 +60,16 @@ def cy_gate(*, dtype="csr"):
     result : :class:`.Qobj`
         Quantum object for operator describing the rotation.
     """
+    dtype = dtype or settings.core["default_dtype"] or _data.CSR
```

Comment:
Just a comment, given that this line is everywhere in qutip core, is there a way to simplify it?